src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionService.kt

```diff
@@ -133,9 +134,13 @@ class SessionService(
   ): List<VisitSessionDto> {
     val firstBookableSessionDay = getFirstBookableSessionDay(requestedBookableStartDate, sessionTemplate.validFromDate, sessionTemplate.dayOfWeek)
     val lastBookableSessionDay = getLastBookableSession(requestedBookableEndDate, sessionTemplate.validToDate)
+    val excludeDates = getExcludeDates(sessionTemplate.prison)
```

Comment:
we could restrict this set to between firstBookableSessionDay and lastBookableSessionDay so that too much data is not returned but accept null as a parameter.

Reply:
same as above.